easy_access/services/merge.py

```
@@ -0,0 +1,301 @@
+            return None
+
+
+def _cast_enum_value(value: Any, enum_class: type) -> Any:
+    """Cast a value to enum, returning the enum value."""
+    if isinstance(value, enum_class):
+        return value.value
+    return value
+
+
+def _cast_values_for_comparison(
+    field: str, new_value: Any, old_value: Any, db_item: Any
```

Ruff (0.14.7)

Dynamically typed expressions (typing.Any) are disallowed in `new_value`

(ANN401)